src/MSBuild.Conversion.Facts/WinUIFacts.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+
+namespace MSBuild.Conversion.Facts
+{
+    /// <summary>
+    ///  A bunch of known values regarding WinUI2 projects.
+    /// </summary>
+    public static class WinUIFacts
+    {
+
+        /// <summary>
+        /// The core set of references WinUI projects use.
+        /// </summary>
+        /// <remarks>
+        /// NEED to update some of these
+        /// </remarks>
+        public static ImmutableArray<string> KnownWinUIReferences => ImmutableArray.Create(
+            "Microsoft.UI.Xaml",
+            "Windows.UI.Xaml",
+            "Microsoft.WinUI",
+            "Microsoft.NETCore.UniversalWindowsPlatform"
+        );
+
+        /// <summary>
+        /// Set of unecessary properties that are ok to remove
+        /// </summary>
+        public static ImmutableArray<string> UnnecessaryProperties => ImmutableArray.Create(
+            "DefaultLanguage",
+            "ProjectGuid", // Are these important? should they be converted instead?
+            "ProjectTypeGuids", //same as above
+            "SubType"
+
+        );
+
+        /// <summary>
+        /// Set of NuGet packages incompatible with WinUI3
+        /// </summary>
+        public static ImmutableArray<string> IncompatiblePackages => ImmutableArray.Create(
+            "Put.Bad.Pkg.Here"
+        );
+
+        public static ImmutableArray<string> WinUIRefs => ImmutableArray.Create(
```

Comment:
nothing will have a reference to `Microsoft.WinUI` since this is the new thing

Reply:
Removed Array now that isWinUI helper method uses TargetPlatformIdentifier instead